R/font-getters.R

```diff
@@ -0,0 +1,136 @@
+#' Get elements of a fontset
+#'
+#' \code{fonts()} returns the list of all \code{font_file} objects for
+#' a given fontset. \code{font()} returns one specific font.
```

Comment:
This is a collection of font families (e.g. Bitstream Vera or Liberation), I'll add a note.